services/src/contexts/mod.rs

```diff
@@ -343,3 +380,85 @@ where
         }
     }
 }
+
+#[cfg(test)]
+
+mod tests {
+    use super::*;
+    use std::path::PathBuf;
+
+    use geoengine_datatypes::{test_data, util::test::TestDefault};
+
+    use crate::{
+        contexts::{Context, InMemoryContext, SimpleSession},
+        util::config::set_config,
+    };
+
+    #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+    async fn read_model_test() {
+        // TODO: find a better solution for this
+        // necessary to avoid getting a wrong config in the other related tests
```

Comment:
just test both in a single test then, until we find a better solution for it

Reply:
now resolved with the `serial_test` crate https://github.com/geo-engine/geoengine/pull/639/commits/15511c1bf666395c74b1bd17f8927fcf0cf0ad1a.